easybuild/framework/extensioneasyblock.py

```diff
@@ -119,9 +119,10 @@ def sanity_check_step(self, exts_filter=None, custom_paths=None, custom_commands
             # unload fake module and clean up
             self.clean_up_fake_module(fake_mod_data)
 
-        if custom_paths or self.cfg['sanity_check_paths'] or custom_commands or self.cfg['sanity_check_commands']:
-            EasyBlock.sanity_check_step(self, custom_paths=custom_paths, custom_commands=custom_commands,
-                extension=self.is_extension)
+        if custom_paths or custom_commands or not self.is_extension:
```

Comment:
because it doesn't make sense, that the one from the parent (there currently no way to specify custom sanity check paths for an extension), and the `sanity_check_step` method being called will pick it up anyway if it needs to (i.e., if `custom_paths` is `None`)